<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogRegApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="LE_Login_Username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>70</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Login_Password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>110</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Login_Password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>113</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Login_Username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Login_Login">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
    color: white; 
    background-color: #2d2d2d; 
    border: 1px solid #333; 
    padding: 5px 10px; 
    font-size: 14px; 
    border-radius: 0px; 
}

QPushButton:hover 
{
    background-color: #333;
    border: 1px solid #555;
}

QPushButton:pressed {
    background-color: #555;
    border: 1px solid #777; 
}
font: 700 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Login_Info">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>230</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Login_Mail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>152</y>
      <width>39</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Почта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Login_Mail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Registration_Username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Registration_Password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>113</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_Registration_Mail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>152</y>
      <width>39</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Почта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NoChangeText_PhoneNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>194</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Номер </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Registration_Username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>70</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Registration_Password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>110</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Registration_Mail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Registration_PhoneNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Registration_Registration">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    color: white; 
    background-color: #2d2d2d; 
    border: 1px solid #333; 
    padding: 5px 10px; 
    font-size: 14px; 
    border-radius: 0px; 
}

QPushButton:hover 
{
    background-color: #333; 
    border: 1px solid #555; 
}

QPushButton:pressed 
{
    background-color: #555; 
    border: 1px solid #777; 
}
font: 700 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Registration_info">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>270</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit 
{
    color: white; 
    border: 1px solid #2b2b2b; 
    padding: 5px; 
}

QLineEdit:focus
 {
    border: 1px solid #524e4e; 
}

QLineEdit::placeholder 
{
    color: gray; 
}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Start_Login">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
    color: white; 
    background-color: #2d2d2d;
    border: 1px solid #333; 
    padding: 5px 10px;
    font-size: 14px; 
    border-radius: 0px;
}

QPushButton:hover
{
    background-color: #333; 
    border: 1px solid #555; 
}

QPushButton:pressed
{
    background-color: #555; 
    border: 1px solid #777; 
}</string>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Start_Registration">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
    color: white;
    background-color: #2d2d2d; 
    border: 1px solid #333; 
    padding: 5px 10px; 
    font-size: 14px; 
    border-radius: 0px; 
}

QPushButton:hover 
{
    background-color: #333; 
    border: 1px solid #555; 
}

QPushButton:pressed 
{
    background-color: #555; 
    border: 1px solid #777; 
}</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Start_Test">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
    color: white; 
    background-color: #2d2d2d;
    border: 1px solid #333; 
    padding: 5px 10px; 
    font-size: 14px; 
    border-radius: 0px; 
}

QPushButton:hover 
{
    background-color: #333; 
    border: 1px solid #555; 
}

QPushButton:pressed 
{
    background-color: #555; 
    border: 1px solid #777; 
}</string>
    </property>
    <property name="text">
     <string>Тест</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
